app.rb

```diff
@@ -1,7 +1,25 @@
 require 'sinatra/base'
+require_relative 'lib/player.rb'
+require_relative 'lib/game.rb'
+
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+
+  get '/' do
+    erb :index
   end
-   run! if app_file == $0
+
+  post '/names' do
+    @player = Player.new(params[:player_name])
+    @game = Game.create(@player)
+    erb :choice
```

Comment:
It's common practise for the response of a post request to be a redirect to a get request that displays the erb file.